tests/test_integration.py

```diff
@@ -409,4 +409,4 @@ def test_update_aggregate_purpose_existing(self):
         manage.update_aggregates()
         db.session.refresh(existing)
         self.assertEqual(existing.total, total + 538)
-        self.assertEqual(existing.count, count + 1)
+        self.assertEqual(existing.count, count + 1)
```

Comment:
Being a bit of a nit-pick here, but could we get a newline at the end of this file just to cut down on linting warnings? Thanks!